src/preamble_minimal.js

```diff
@@ -65,17 +65,17 @@ function updateGlobalBufferAndViews(b) {
 #if SUPPORT_BIG_ENDIAN
   HEAP_DATA_VIEW = new DataView(b);
 #endif
-  HEAP8 = new Int8Array(b);
-  HEAP16 = new Int16Array(b);
-  HEAP32 = new Int32Array(b);
-  HEAPU8 = new Uint8Array(b);
-  HEAPU16 = new Uint16Array(b);
-  HEAPU32 = new Uint32Array(b);
-  HEAPF32 = new Float32Array(b);
-  HEAPF64 = new Float64Array(b);
+  Module['HEAP8'] = HEAP8 = new Int8Array(b);
+  Module['HEAP16'] = HEAP16 = new Int16Array(b);
+  Module['HEAP32'] = HEAP32 = new Int32Array(b);
+  Module['HEAPU8'] = HEAPU8 = new Uint8Array(b);
+  Module['HEAPU16'] = HEAPU16 = new Uint16Array(b);
+  Module['HEAPU32'] = HEAPU32 = new Uint32Array(b);
+  Module['HEAPF32'] = HEAPF32 = new Float32Array(b);
+  Module['HEAPF64'] = HEAPF64 = new Float64Array(b);
 #if WASM_BIGINT
-  HEAP64 = new BigInt64Array(b);
-  HEAPU64 = new BigUint64Array(b);
+  Module['HEAP64'] = HEAP64 = new BigInt64Array(b);
+  Module['HEAPU64'] = HEAPU64 = new BigUint64Array(b);
```

Comment:
Ditto, the assignments to Module should only be needed in the MODULARIZE case.

Reply:
Looking at `preamble.js`, it happens no matter if it's a module or not. So I guess we should do the same?

Reply:
I believe one of main differences with MINIMAL_RUNTIME is that it does away with the Module object by default.  To save space.   With MODULARIZE however, we are force to revert to exporting everything on that object.